com.unity.ml-agents/Editor/BehaviorParametersEditor.cs

```diff
@@ -24,25 +25,48 @@ public override void OnInspectorGUI()
             so.Update();
 
             // Drawing the Behavior Parameters
-            EditorGUI.BeginChangeCheck();
             EditorGUI.indentLevel++;
+            EditorGUI.BeginChangeCheck();
             EditorGUILayout.PropertyField(so.FindProperty("m_BehaviorName"));
-            EditorGUILayout.PropertyField(so.FindProperty("m_BrainParameters"), true);
+            bool needModelUpdate = EditorGUI.EndChangeCheck();
+
+            EditorGUI.BeginChangeCheck();
+            EditorGUI.BeginDisabledGroup(Application.isPlaying);
+            {
+                EditorGUILayout.PropertyField(so.FindProperty("m_BrainParameters"), true);
+            }
+            EditorGUI.EndDisabledGroup();
+            var brainParamsChanged = EditorGUI.EndChangeCheck();
+
+            EditorGUI.BeginChangeCheck();
```

Comment:
not that you have to do this, but you can also scope with braces if the indentation would make it more readable.

Reply:
Yeah, I have that in some of the other editors (namely for the disable groups). I'll pull the `*changed` vars to bools at the top of the code.